airflow/www/static/js/main.js

```diff
@@ -286,4 +286,8 @@ $(document).ready(() => {
 
   // Global Tooltip selector
   $(".js-tooltip").tooltip();
+
+  // Turn off autocomplete for login form
+  $("#username:input")[0].autocomplete="off";
```

Comment:
Maybe we should do this only for password? @pierrejeambrun

Reply:
I thought about that, but I think it also prevent the browser from caching and saving stuff locally, so I think we need to keep it. (doesn't hurt)

Reply:
I was just trying to be consistent. The issue only mentions password, but I didn't think it would hurt to apply the change to username too. Then it's even more "secure" from the issue's perspective I think.

Shall I apply this change on `main` and reopen the PR there and close this one?

Reply:
You can close and re-open yes, or just change the target branch of the PR and rebase your commits on top of main

Reply:
I have recreated the PR in `main` at https://github.com/apache/airflow/pull/44929. Closing this PR.